app/api/admin_statistics_api/users.py

```diff
@@ -53,17 +53,28 @@ def get_all_user_roles(self, role_name):
             Event.deleted_at.is_(None), UsersEventsRoles.role == role)
         return uers
 
+    def get_distinct_roles_count(self, query_result):
+        """
+        Returns the length of set of users for the given query
+        """
+        distinct_entry = set([i.user for i in query_result])
+        return len(distinct_entry)
+
     def organizer_count(self, obj):
-        return get_count(self.get_all_user_roles('organizer'))
+        query_result = self.get_all_user_roles('organizer')
+        return self.get_distinct_roles_count(query_result)
```

Comment:
Very unoptimized approach. The query itself should use distinct count. Not load up every user in memory and then add in a set

Please do the same for others as well

Reply:
@iamareebjamal Understood, will try and improve it in the next commmit

Reply:
@iamareebjamal Please check out the query now.